Missing required positional argument in run method

There is a bug in the test's helper method `run` which expects 1 required positional argument `entrypoint`. There should be a default value set to `False` because the method is also called with `entrypoint` argument omitted:

```python
hlp_m = run('--help')
hlp_e = run('--help', entrypoint=True)
```

It leads to an error in module tests:

```
def test_incorrect_config(config_name):
        """And if they don't forget, it still may be incorrect"""
>       cp = run(f'--config "{config(config_name)}" '
                 f'something/dummy')
E       TypeError: run() missing 1 required positional argument: 'entrypoint'
```

Possible solution is:

```python
def run(line, entrypoint=False, **kwargs):
   pass
```

--- a/tests/helper.py
+++ b/tests/helper.py
@@ -28,7 +28,7 @@ def env(**kwargs):
                 os.environ[key] = value
 
 
-def run(line, entrypoint, **kwargs):
+def run(line, entrypoint=False, **kwargs):
     if entrypoint:
         print('$ committee', line)
         command = ['committee']

